### PR TITLE
feature/apr-2024-api-update

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
## PR Overview
**This PR will address the following Issue/Feature:**
- https://github.com/fivetran/dbt_facebook_pages_source/issues/10

**This PR will result in the following new package version:**

- v0.3.0: breaking due to schema changes in the source (deleting columns)

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

> ## 🚨 Breaking Changes 🚨:
> - This change is made breaking since columns have been removed in the source package (see the [dbt_facebook_pages_source v0.3.0 CHANGELOG](https://github.com/fivetran/dbt_facebook_pages_source/blob/main/CHANGELOG.md#dbt_facebook_pages_source-v030) for more details). 
>     - No columns were removed from the end models in this package, however if you use the staging models independently, you will need to update your downstream use cases accordingly.
>     - Columns removed from staging model `stg_facebook_pages__daily_page_metrics_total`:
>         - `consumptions`
>         - `content_activity`
>         - `engaged_users`
>         - `places_checkin_mobile`
>         - `views_external_referrals`
>         - `views_logged_in_total`
>         - `views_logout`
>     - Columns removed from staging model `stg_facebook_pages__lifetime_post_metrics_total`:
>         - `impressions_fan_paid`
> 
> ## Features
> - Updated documentation to reflect the current schema. 
> 
> ## Under the Hood:
> - Incorporated the new `fivetran_utils.drop_schemas_automation` macro into the end of each Buildkite integration test job.
> - Updated the pull request templates.
> - Included auto-releaser GitHub Actions workflow to automate future releases.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [ ] ~dbt run (if incremental models are present) && dbt test~

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

- Columns have been removed in the source but do not change the output of this package.  

### If you had to summarize this PR in an emoji, which would it be?
 
📄

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 packages:
-- package: fivetran/facebook_pages_source
-  version: [">=0.2.0", "<0.3.0"]
\ No newline at end of file
+# - package: fivetran/facebook_pages_source
+#   version: [">=0.3.0", "<0.4.0"]
+- git: https://github.com/fivetran/dbt_facebook_pages_source.git
+  revision: feature/apr-2024-api-update
+  warn-unpinned: false
```
Comment (fivetran-catfritz) [inline packages.yml:6]:
```suggestion
- package: fivetran/facebook_pages_source
  version: [">=0.3.0", "<0.4.0"]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
+        - `content_activity`
+        - `engaged_users`
+        - `places_checkin_mobile`
+        - `views_external_referrals`
+        - `views_logged_in_total`
+        - `views_logout`
+    - Columns removed from staging model `stg_facebook_pages__lifetime_post_metrics_total`:
+        - `impressions_fan_paid`
+
+## Features
```
Comment (fivetran-avinash) [inline CHANGELOG.md:18]: It doesn't look like schema documentation was updated in this package, so this can be removed.

File: packages.yml (hunk 1 above)
Comment (fivetran-avinash) [reply packages.yml:6]: Oh nice way to handle this!

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-catfritz One small request but otherwise lgtm!

**Pull request merged**